src/rollback_to_stable/rts_btree_walk.c

```diff
@@ -75,6 +75,11 @@ __rts_btree_walk_page_skip(
             *skipp = true;
         }
         WT_REF_SET_STATE(ref, WT_REF_DELETED);
+        __wt_verbose_level_multi(session, WT_VERB_RECOVERY_RTS(session), WT_VERBOSE_DEBUG_3,
+          WT_RTS_VERB_TAG_PAGE_DELETE
+          "deleted page with durable_timestamp=%s > rollback_timestamp=%s",
+          __wt_timestamp_to_string(page_del->durable_timestamp, time_string),
```

Comment:
Add the page_del `transaction id` and also the `commit timestamp` here.

Reply:
Done in: [2ff1584](https://github.com/wiredtiger/wiredtiger/pull/8960/commits/2ff15849792bdb0a1de4f07f3f8cb187db8e72a2)